sentry/src/main/java/io/sentry/util/LazyEvaluator.java

```diff
@@ -48,6 +48,12 @@ public void setValue(final @Nullable T value) {
     }
   }
 
+  public void resetValue() {
```

Comment:
Can you update the javadoc for `getValue()`? right now it says
> Executes the evaluator function and caches its result, so that it's called only once.

How about something like this:
> Returns the value provided by the evaluator function. The value is cached internally and will be returned in subsequent calls, unless resetValue is called.